File: testprojects/java/benchmark/accuracy-test-2013-04-17/domain/indirect/allowedfrom/CallInstanceMethodIndirect_VarMethod.java
package domain.indirect.allowedfrom;

import domain.indirect.intermediate.BackgroundService;

public class CallInstanceMethodIndirect_VarMethod {
	private BackgroundService bckg;
	private String test;
	
	public CallInstanceMethodIndirect_VarMethod() {
		bckg = new BackgroundService();
	}

	// Indirect access via normal method 
	public String case2(){
		test = bckg.serviceOne.getName();
		return test;
	}
}

File: testresources/csharp/benchmark/Domain/Indirect/AlowedFrom/CallInstanceMethodIndirect_VarMethod.cs
namespace Domain.Indirect.AllowedFrom
{
	public class CallInstanceMethodIndirect_VarMethod
	{
        private Domain.Indirect.Intermediate.BackgroundService bckg;

		private string test;
 
		public void Case2()
		{
            bckg = new Domain.Indirect.Intermediate.BackgroundService();
			test = bckg.serviceOne.GetName();
		}
	}
}
